### PR TITLE
Update What-is-Gas.md and What-is-Mining.md Regarding the ETH 2.0 or Eth2 misnomer and Post Merge edits.

### PR DESCRIPTION
**Pull request by RV12R**
As the roadmap for Ethereum has evolved, Ethereum 2.0 has become an inaccurate representation of Ethereum’s roadmap. It is also misrepresented by some as Etheruem after the merge.This is clarified in their [official blog](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming).

### CODE REVIEW

**Comment by RV12R** [comment]
Changing "What-is-Mining" to Before the merge and After the Merge (EIP 3675) can be done. Comment on this

File: What-is-Mining.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 Mining is the process that helps create new blocks of transactions to be added to the Ethereum blockchain network. It also helps keep the network secure from attacks.
 
-Just like Bitcoin, Ethereum currently uses a Proof-of-Work (PoW) consensus mechanism, which is heavily reliant on mining. Ethereum miners use their time and computation power to process new incoming transactions and produce new blocks.
+Just like Bitcoin, Ethereum was using a Proof-of-Work (PoW) consensus mechanism, which is heavily reliant on mining. Ethereum miners used their time and computation power to process new incoming transactions and produce new blocks untill Ethereum upgraded to Proof-of-Stake consensus mechanism (PoS).
 
-> Ethereum 2.0 (scheduled for mid-late 2022) will replace Proof of Work with Proof of Stake, which will significantly change how new blocks are produced. We will talk about Eth2 later.
+> [Ethereum Merge](https://ethereum.org/en/upgrades/merge/) (The Merge was executed on September 15, 2022) it replaced Proof of Work with Proof of Stake, which significantly changed how new blocks are produced reducing energy consumption by ~99.95%. We will talk more about Proof-of-Stake later.
```
Comment (yanukadeneth99) [inline What-is-Mining.md:7]: Nice!

**Review by yanukadeneth99 — APPROVED** [review]

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]

**Comment by RV12R** [comment]
Changes made check it out

**Comment by yanukadeneth99** [comment]
Hey @RV12R, great change! Thank you.

**Pull request merged**